Gemfile

```diff
@@ -18,7 +18,7 @@ gem "html-pipeline"
 gem "kramdown"
 gem "mdl"
 
-gem "govuk_publishing_components", "~> 24.4"
+gem "govuk_publishing_components"
```

Comment:
This was introduced [here](https://github.com/alphagov/govuk-developer-docs/commit/8bacd1d9ea6048d0a8034f5a4c9e1111bf6abe8b), but I don't understand the explanation. Seems reasonable to remove.

Reply:
Interesting. Running `bundle install` will just pull what's from the Gemfile.lock and won't impact it.

The situation where it'd not default to installing the newest version with a `bundle update` is if there was a shared dependency of this app and govuk_publishing_components where there were version constraints. Bundler then needs to negotiate what it thinks is the appropriate version (for example if current version of govuk_publishing_components depended on an old version of Kramdown, and an older version had an unbounded dependency it might go for an old govuk_publishing_components to satisfy a new Kramdown) - these situations seem to be rather rare though.

I've just tried running a full `bundle update` to see if it downgrades, but it was taking ages so I'll check that on Monday.

Reply:
I found out - after leaving it running for ages (it was running for hours) - that the problem is because [govuk_tech_docs gem specifies sprockets 4](https://github.com/alphagov/tech-docs-gem/blob/master/govuk_tech_docs.gemspec#L49) or greater, so I guess it tries to find a govuk_publishing_components without the constraint. It turns out that constraint in govuk_tech_docs isn't needed so I'm going to open a PR to remove it.

Reply:
Oh that's interesting - nice catch! I guess that might explain the excessive run time.